ext/afform/core/managed/AfformTags.mgd.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+use CRM_Afform_ExtensionUtil as E;
+
+// This file:
+// - adds option value to `tag_used_for`, allowing Afforms to be tagged
+//   NOTE: this is unusual in that afform is not a db entity
+//   (normally the value normally corresponds to a physical table name in the
+//   database - here it is just the entity name)
+//
+// - creates an example tag that can be used for afforms. This is more to demonstrate
+//   how it can be used, as Donor Journey might be useful for some orgs but not for others
+//   @todo think of better example tag that is more universally applicable?
```

Comment:
I think it's better not.

As a general principle I think we should not install sample data onto people's sites unless they specifically select the "sample data" option in the installer.

Reply:
General principle: definitely agree.

My worry here though is that the "Used For" on Tags isn't discoverable enough, so people might not find how to add tags that they can use here, so I thought this might be a reasonable exception to the rule. (Note we add example Tags by default on new installs. Also Tags are by their nature very easy to throw away, so it feels not such an imposition to add this one?)

But I don't mind, can remove if you're not convinced.

Reply:
Yea, I had the same concern when I added tagging to SearchKit, so my solution was to create the select-or-create tag widget inspired by GitHub tags (try tagging a searchKit, it's cool).
We could probably extract that widget from SearchKit and reuse it here.

Reply:
Ooh I hadn't seen that before. Definitely a useful widget! For formbuilder too please :wink: 

Component will definitely need a bit of refactoring because the component is using `EntityTag` api... but would be great definitely. Happy to remove this in the meantime

(Slightly separate point - I find different orgs have different policies on whether inline tag creation is a useful feature or makes it too easy for thousands of unhelpful tags to proliferate. So might be good if there was some way to configure that... At the moment for quickform tag widgets it seems to vary a bit on different screens...)